### DISCUSSION TITLE
Release frequency / timeline for `1.0.1` release

### DISCUSSION THREAD

**Discussion by valpesendorfer** [duckdb/duckdb#13739] [General]
Hello,

I was wondering if `duckdb` has a defined release cadence, and at which time horizon we're looking while waiting for the release of `1.0.1` which includes a few fixes that are important for us.

Thank you!

**Comment by carlopi** [comment]
Release calendar is at https://duckdb.org/docs/dev/release_calendar#upcoming-releases, where you will find that v1.1.0 is planned for next week.

At some point I had a duckdb query to find that, might eventually add it (as a community extension?)

**Reply by valpesendorfer** [reply]
thanks!!! I totally missed that, apologies.
